packages/lesmis/src/components/MiniMap.js

```diff
@@ -24,14 +25,38 @@ const Map = styled(MapContainer)`
   height: auto;
 `;
 
+const BasicPolygon = styled(Polygon)`
+  fill: ${RED};
+  fill-opacity: 0.3;
+  stroke: ${RED};
+`;
+
+/* eslint-disable react/prop-types */
+const CountryMask = ({ countryData }) => {
+  return <InversePolygonMask region={countryData?.region} />;
+};
+
+const RegionPolygon = ({ entityData }) => {
+  if (!entityData?.region) return null;
+  if (entityData?.type === 'country') return null; // country is fine without, as it has the mask
+
+  return <BasicPolygon positions={entityData?.region} interactive={false} />;
+};
+
+const PointMarker = ({ entityData }) =>
+  entityData?.point && <IconMarker coordinates={entityData?.point} color={RED} scale={1.5} />;
+/* eslint-enable react/prop-types */
+
 export const MiniMap = ({ entityCode }) => {
+  const { data: countryData, isLoading: isLoadingCountryData } = useEntityData(COUNTRY_CODE);
   const { data: entityData, isLoading: isLoadingEntityData } = useEntityData(entityCode);
 
-  return isLoadingEntityData ? null : (
+  return isLoadingCountryData || isLoadingEntityData ? null : (
     <Map bounds={entityData?.bounds} dragging={false} zoomControl={false}>
       <TileLayer tileSetUrl={TILE_SET_URL} />
```

Comment:
I would vote for not-showing the attribution and mapbox link on the mini-map because I think it looks messy. I feel that since we are showing it on the main map we could get away with it. It could be turned off by a hidden prop or we could just use the leaflet tilelayer directly which doesn't have it.

Reply:
It does look messy!

Reply:
Removed, though it needed css applied at the MapContainer level because of the way leaflet structures the DOM.